client/PublicRequestTypes.ts

```diff
@@ -456,8 +456,8 @@ export interface SignMessageRequest extends BasicRequest {
 
 export interface SignedMessage {
     signer: string; // Userfriendly address
-    signerPublicKey: Uint8Array;
-    signature: Uint8Array;
+    signerPublicKey: number[];
+    signature: number[];
```

Comment:
Should state which values are to be expected here, i.e. integers in the range 0..255.